src/components/HelFormFields/HelLanguageSelect.js

```diff
@@ -20,7 +21,7 @@ class HelLanguageSelect extends React.Component {
         let checked = _.filter(this.refs, (ref) => (ref.getChecked()))
```

Comment:
I think this method should use the constructor bind style as well, instead of using arrow function in render.

Reply:
It's ok to leave it like that, arrow function might not good but it's ok